src/SprykerEco/Zed/AfterPay/Communication/Plugin/Checkout/AfterPayPostCheckPlugin.php

```diff
@@ -21,7 +23,7 @@
 class AfterPayPostCheckPlugin extends AbstractPlugin implements CheckoutPostSaveHookInterface
 {
     /**
-     * Specification:
+     * {@inheritDoc}
      * - This plugin is called after the order is placed.
      * - Set the success flag to false, if redirect should be headed to an error page afterwords
```

Comment:
Please, add  `.` Set the success flag to false, if redirect should be headed to an error page afterwords.